### DISCUSSION TITLE
How to display <Stories /> with expanded code preview on default?

### DISCUSSION THREAD

**Discussion by struers-gmo** [storybookjs/storybook#29045] [Help] [answered]
### Summary

Hello,
Is it possible to display a Doc Block <Stories /> expanded on default? I mean the "Show code" option.

![image](https://github.com/user-attachments/assets/efa5f250-2fe9-49ec-af59-7965411c6a8a)

Thank you:)

**Comment by smlimasu** [comment, marked as answer]
parameters: { 
docs: { 
canvas: { 
sourceState: 'shown'
}}}

Apply in preview.ts for all stories or at the meta or story level for individual configuration

**Reply by struers-gmo** [reply]
Thank you, works:)
